api/progress.go

```diff
@@ -0,0 +1,82 @@
+package api
+
+import (
+	"TUM-Live/dao"
+	"TUM-Live/tools"
+	"github.com/gin-gonic/gin"
+	"net/http"
+)
+
+func configProgressRouter(router *gin.Engine) {
+	router.POST("/api/progressReport", saveProgress)
+	router.POST("/api/progressRequest", fetchProgress)
```

Comment:
Semantically this should be a GET request, but I think the progress should be fetched when the site is loaded (`GET /w/GBS/123`) and passed to your videjs plugin through the DOM
This saves an extra HTTP request and there won't be any weird jumps in case the request is delayed.

Reply:
This makes sense :+1: Better loading times and less traffic is always good.